src/Mvc/Mvc.ViewFeatures/src/Infrastructure/UnsupportedJavaScriptRuntime.cs

```diff
@@ -13,10 +13,5 @@ public Task<T> InvokeAsync<T>(string identifier, params object[] args)
         {
             throw new InvalidOperationException("JavaScript interop calls cannot be issued during server-side prerendering, because the page has not yet loaded in the browser. Prerendered components must wrap any JavaScript interop calls in conditional logic to ensure those interop calls are not attempted during prerendering.");
         }
-
-        public void UntrackObjectRef(DotNetObjectRef dotNetObjectRef)
```

Comment:
dead code? Isn't this called by interop?

Reply:
It's being removed in https://github.com/aspnet/Extensions/pull/1704